gax-java/gax-grpc/src/test/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProviderTest.java

```diff
@@ -877,6 +879,43 @@ public void canUseDirectPath_nonGDUUniverseDomain() {
     Truth.assertThat(provider.canUseDirectPath()).isFalse();
   }
 
+  @Test
```

Comment:
Right I could expose the headers from GrpcHeaderInterceptor, but GrpcHeaderInterceptor is only exposed from a grpc package private ManagedChannelOrphanWrapper, so I would also have to make GrpcHeaderInterceptor a member variable and expose it from InstantiatingGrpcChannelProvider, but that seemed too specific for unit testing so instead I chose to cover it in the IT tests.  Let me know if you think it would be better to expose the variable and cover it in unit